t3f/ops.py

```diff
@@ -790,32 +790,42 @@ def multiply(tt_left, right):
 
 
 def frobenius_norm_squared(tt, differentiable=False):
-  """Frobenius norm squared of a TensorTrain (sum of squares of all elements).
+  """Frobenius norm squared of `TensorTrain` or of each TT in `TensorTrainBatch`.
 
   Args:
```

Comment:
One final thing (sorry)): since you removed "sum of squares of all elements" part, can you please add it to the next line while leaving one blank line?
"""Frobenius norm squared of `TensorTrain` or of each TT in `TensorTrainBatch`.

Frobenius norm squared is the sum of squares of all elements in a tensor.

Args:
...
"""